build.fsx

```diff
@@ -1,32 +1,24 @@
-#r "paket:
-nuget BlackFox.Fake.BuildTask
-nuget Fake.Core.Target
-nuget Fake.Core.Process
-nuget Fake.Core.ReleaseNotes
-nuget Fake.IO.FileSystem
-nuget Fake.DotNet.Cli
-nuget Fake.DotNet.MSBuild
-nuget Fake.DotNet.AssemblyInfoFile
-nuget Fake.DotNet.Paket
-nuget Fake.DotNet.FSFormatting
-nuget Fake.DotNet.Fsi
-nuget Fake.DotNet.NuGet
-nuget Fake.Api.Github
-nuget Fake.DotNet.Testing.Expecto //"
```

Comment:
From my personal experience i had neither problems with inline dependencies nor with a references build-group. It's just that i think the build-group looks cleaner, but i understand your convention and changed it back. 
With the "+BuildTask DSL" you kinda lost me, i am not sure what you mean.

Reply:
Its a DSL (Domain Specific Language) for FAKE build targets. It makes dependencies between more obvious and cleaner, because they are typed instead of being just the string trees at the end of the file (https://github.com/vbfox/FoxSharp/blob/master/src/BlackFox.Fake.BuildTask/Readme.md). I will add this after the PR is merged, as ther ewill be many small changes in the build script